Code/GraphMol/Fingerprints/FingerprintUtil.h

```diff
@@ -79,6 +79,9 @@ RDKIT_FINGERPRINTS_EXPORT std::uint64_t getTopologicalTorsionCode(
 
 RDKIT_FINGERPRINTS_EXPORT std::uint32_t getTopologicalTorsionHash(
     const std::vector<std::uint32_t> &pathCodes);
+
+RDKIT_FINGERPRINTS_EXPORT unsigned int numPiElectrons(const Atom *atom);
```

Comment:
Having this be in the public API in `Fingerprints` seems a bit strange. How would you feel about moving it to `MolOps` or somewhere like that?